python/pyspark/sql/tests/test_arrow.py

```diff
@@ -118,32 +117,50 @@ def create_pandas_data_frame(self):
         data_dict["4_float_t"] = np.float32(data_dict["4_float_t"])
         return pd.DataFrame(data=data_dict)
 
-    def test_toPandas_fallback_enabled(self):
+    # def test_toPandas_fallback_enabled(self):
```

Comment:
Can you enable the tests back?

Reply:
done. However i have changed the expected failure type to Map<TimeStamp,TimeStamp> instead of ArrayType<TimeStamp> . Since ArrayType<TimeStamp> will not throw exception